index.html

```diff
@@ -3576,7 +3576,7 @@ <h5>JWT Encoding</h5>
 
             <p>
 For backward compatibility with JWT processors, the following registered JWT
-claim names MUST be used instead of their respective standard 
+claim names MUST be used instead of, or in addition to, their respective standard 
```

Comment:
```suggestion
claim names MUST be used, instead of or in addition to, their respective standard 
```

Reply:
@David-Chadwick -- You marked this resolved, and said that such marking meant you had applied my suggested change.  This change [was not applied](https://github.com/w3c/vc-data-model/blob/e77940a0309cc43a1b198fc3e0769c449c25f796/index.html#L3579).  I am applying it now in a new PR.